File: employ.py
#!/usr/bin/python3
"""pick where to post employees"""


def postings():
    """post employees based on sex, age or marital status"""
    age = int(input("Enter the age of employee: "))
    if age < 20 or age > 60:
        print("ERROR")
    sex = input("What is your sex? 'M' or 'F': ")
    marital = input("Are you married? 'Y' or 'N': ")
    if sex == 'F':
        print("You will work in urban areas")
    elif age >= 20 and age <= 40:
        print("You will work anywhere")
    elif age > 40 and age <= 60:
        print("You will work in urban areas")


if __name__ == "__main__":
    postings()

File: employ.java
import java.util.Scanner;

public class employ {
    public static void main(String[] args){
        /*post employees by their age and gender */
        Scanner myScanner = new Scanner(System.in);
        System.out.println("Are you male or female? Enter 'M' or 'F': ");
        String input = myScanner.nextLine();
        char s = input.charAt(0);
        System.out.println("Are you married? Answer: 'Y' or 'N': ");
        input = myScanner.nextLine();
        //char m = input.charAt(0);
        System.out.println("How old are you? ");
        int age = myScanner.nextInt();
        myScanner.close();
        if (age < 20){
            System.out.println("You are underage");
        }
        else if (age > 60){
            System.out.println("Please retire");
        }
        else if (s == 'F' || age >= 40){
            System.out.println("You will work in urban areas");
        }
        else if (age >= 20 && age < 40){
            System.out.println("You will work anywhere");
        }
    }
}
